src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -276,6 +278,12 @@
      */
     private boolean callMaxOccupantsLimitReached = false;
 
+    /**
+     * startMutedFlags[0] represents the muted status of audio stream.
+     * startMuted[1] represents the muted status of video stream.
+     */
+    private boolean[] startMuted = { false, false };
```

Comment:
Do we need to handle the video muted stuff as jigasi is audio only and we care only for audio?

Reply:
For video the flag is ignored but wanted to be consistent with startmuted extension.

Reply:
Yeah, but removing it will significantly simplify code.

Reply:
Removed the flag for the video stream